mmv1/templates/terraform/extra_schema_entry/vertex_ai_feature_group_feature.go.erb

```diff
@@ -0,0 +1,5 @@
+"region": {
+	Type:			schema.TypeString,
+	Computed: 		true,
```

Comment:
nit: can you clean up the spacing here so that internal alignment is done w/ spaces?